app/mailers/jobseekers/vacancy_mailer.rb

```diff
@@ -1,9 +1,10 @@
 class Jobseekers::VacancyMailer < Jobseekers::BaseMailer
   def draft_application_only(job_application)
     @job_application = job_application
+    @jobseeker = job_application.jobseeker
     @vacancy = job_application.vacancy
 
-    send_email(to: job_application.email_address,
```

Comment:
I think we need to do this for 'closed_early_ emails too - they seem to be the main source of the sentry error